### PR TITLE
Update the vSMP userdata

### PR DESCRIPTION
**Pull request by hardikdr**
**How to categorize this PR?**

/area usability
/kind enhancement
/priority normal
/os suse-chost

**What this PR does / why we need it**: Avoid downloading vSMP MemoryOne packages via user-data. 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
Updated vSMP MemoryOne OS userdata to not download vSMP packages.
```

### CODE REVIEW

**Comment by hardikdr** [comment]
The verify step is failing on:
```
Failed to install helm with the arguments provided: --version v2.13.1
Accepted cli arguments are:
	[--help|-h ] ->> prints this help
	[--version|-v <desired_version>] . When not defined it defaults to latest
	e.g. --version v2.4.0  or -v latest
	[--no-sudo]  ->> install without sudo
	For support, go to https://github.com/helm/helm.
```

It doesn't seem to be related to the proposed changes in the PR. Any ideas, how to fix it?

**Comment by tim-ebert** [comment]
You need something similar to this: https://github.com/gardener/gardener-extension-os-ubuntu-alicloud/pull/10

**Comment by hardikdr** [comment]
@vpnachev @vlvasilev Should we fix the same here, if you agree, I can copy that PR on this repo as well.

**Comment by vpnachev** [comment]
It would be better to vendor g/g@v1.7.x

**Comment by hardikdr** [comment]
Okay, I'll then vendor gardener-1.7.0 , and fix the failing tests.

**Comment by hardikdr** [comment]
/hold
Ready for review, but I would like to test a bit more.

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -20,8 +20,7 @@ If multiple identifiers make sense you can also state the commands multiple time
 -->
 /area TODO
 /kind bug
-/priority normal
-/os suse-chost
+/priority normaln/os suse-chost
```
Comment (hardikdr) [inline .github/ISSUE_TEMPLATE/bug.md:23]: This seems to be changed as a result of `make revendor`, not sure why.

**Comment by hardikdr** [comment]
I tested it locally, seems to work fine. Though, I see, due to the name change to `chost`, there are a couple of configurations needed in terms of rbac and controller-registrations while running with g/g.

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
Comment (tim-ebert) [reply .github/ISSUE_TEMPLATE/bug.md:23]: Please follow `make install-requirements` to install `gnu-sed` and re-run `make revendor`.
I think `sed` in `hack/update-github-templates.sh` is assuming `gnu-sed`, so please add it to your path accordingly :)

**Comment by hardikdr** [comment]
Thanks, @tim-ebert  for the suggestion, I used gsed for updating the templates, seemed to work.

**Comment by hardikdr** [comment]
@gardener/gardener-extension-os-suse-chost-maintainers  @gardener/gardener-maintainers  Will it please be possible to cut the release of this component. We have a stakeholder waiting for this change to be consumed as soon as possible.

**Comment by tim-ebert** [comment]
@vpnachev or @vlvasilev Would one of you like to take over this task?

**Comment by vpnachev** [comment]
It is released - https://github.com/gardener/gardener-extension-os-suse-chost/releases/tag/v1.8.0

**Pull request merged**